### PR TITLE
Blog app: Validation and specs

### PR DESCRIPTION
**Pull request by manzitresor**
### Implemented Feature

**Added validations:**

- [ ] For the User model:

  -Name must not be blank.
  -PostsCounter must be an integer greater than or equal to zero.
  -For the Post model:
  -Title must not be blank.
  -Title must not exceed 250 characters.
  -CommentsCounter must be an integer greater than or equal to zero.
  -LikesCounter must be an integer greater than or equal to zero.

- [ ] Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: spec/model/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tresor', photo: 'hhtps/manzi.com', bio: 'this is manzi user', postsCounter: 2) }
```
Comment (Owusu-Desmond) [inline spec/model/user_spec.rb:4]: - [ ] Please ensure that any custom methods in the models have a corresponding unit test, So make sure you add a unit test for the `recent_posts` method in your user model :blush:

File: spec/model/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```
Comment (Owusu-Desmond) [inline spec/model/post_spec.rb:3]: - [ ] Please ensure that any custom methods in the models have a corresponding unit test, So make sure you add a unit test for the `recent_comments` method in your post model :blush:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @manzitresor , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 All tests passed ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @manzitresor 👏 👏 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Implemented tests for` recent_post` and `recent_comment` ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**